Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM maven:3.6.3-jdk-13 as build_jar
+RUN yum install git -y
+WORKDIR /jar
+COPY pom.xml .
+RUN mvn dependency:go-offline
+COPY src/ /jar/src/
+RUN mvn package
```

Comment:
>Then we can build it like docker build --build-arg="version=${BUILD_TAG}" and git tag will be propagated to jar manifest.

Created a TODO for that

> Also, I'd use -B and --quiet options for mvn package, since most probably build will be called from CD tools.

Done